File: 17608.py
import sys

N = int(sys.stdin.readline())
sticks = []
for i in range(N):
    sticks.append(int(sys.stdin.readline()))

cnt = max = 0
while len(sticks) > 0:
    a = sticks.pop()
    if a > max:
        cnt += 1
        max = a

print(cnt)

File: 17608.cpp
#include <iostream>
using namespace std;

int sticks[100000];

int main() {
	int N; cin >> N;
	for (int i = 0; i < N; i++)
		cin >> sticks[i];

	int cnt = 0, max = 0;
	for (int i = N - 1; i >= 0; i--) {
		if (sticks[i] > max) {
			cnt++;
			max = sticks[i];
		}
	}

	cout << cnt << endl;
}
